.github/workflows/reusable_build.yml

```diff
@@ -0,0 +1,198 @@
+name: Reusable build
+on:
+  workflow_call:
+    inputs:
+      runs_on:
+        description: Runner to run the job on
+        required: true
+        type: string
+        default: heavy
+
+      container:
+        description: "The container object as a JSON string (leave empty to run natively)"
+        required: true
+        type: string
+        default: ""
+
+      conan_profile:
+        description: Conan profile to use
+        required: true
+        type: string
+
+      build_type:
+        description: Build type
+        required: true
+        type: string
+
+      disable_ccache:
+        description: Whether ccache should be disabled
+        required: false
+        type: boolean
+        default: false
+
+      conan_cache:
+        description: Whether to use conan cache
+        required: false
+        type: boolean
+        default: true
```

Comment:
I think we could have only one option `disable_cache`. I couldn't imagine a situation where we are ok with one cache, but want to avoid another one. WDYT?

Reply:
I can remove second cache input.